docs/mining/proof-of-stake.md

```diff
@@ -26,6 +26,30 @@ You will also need to [Obtain DCR](/getting-started/obtaining-dcr.md) if you hav
 
 ---
 
+## ** How Does It Work? ** ##
+
+Purchasing a ticket for PoS is quite simple (see below) but what happens to it after you buy it?
+A ticket on main net (test net uses different parameters) will go through a few stages in its lifetime:
+
+1. You buy a ticket using your wallet.
+2. The ticket enters the 'mempool'. Only 20 tickets are mined into each block. This is where tickets wait to be mined.
+3. Tickets are mined into a block in order of their `ticketfee`. Note that the ticketfee is priced per KB. 
+A ticket transaction is about a third (solo) or half (pool) a KB so the actual fee you pay will be scaled accordingly.
+4. **A -** If your ticket is mined into a block, it will enter immature status for 256 blocks (about 20 hours). During
+this time the ticket cannot vote. At this point, the ticket fee is non-refundable.  
+**B -** If your ticket is not mined, both the ticket price and fee are returned.
```

Comment:
*ticket* fee is returned, but the fee for any regular tx that was created to be used for ticket inputs will be gone if that regular tx was mined. (what we call the split tx)